topics/single-cell/tutorials/scrna-case_monocle3-rstudio/tutorial.md

```diff
@@ -36,6 +36,13 @@ requirements:
         - scrna-case_alevin-combine-datasets
         - scrna-case_basic-pipeline
 
+follow_up_training:
+-
+    type: "internal"
+    topic_name: single-cell
+    tutorials:
+        - EBI-retrieval
```

Comment:
@nomadscientist why is "Importing files from public atlases" coming as a followup? I would assume as a first step?

Reply:
Ahh, well no - scientists usually/often come with their own data, so this was my way of pointing people at also using public resources (like EBI single-cell expression atlas).